### PR TITLE
Add drf Request support

### PR DESCRIPTION
**Pull request by Niccolum**
## Related issues
closes #ISSUE

### CODE REVIEW

File: elasticapm/contrib/django/client.py [hunk 1]
```diff
@@ -36,6 +36,10 @@
 from django.core.exceptions import DisallowedHost
 from django.db import DatabaseError
 from django.http import HttpRequest
+try:
+    from rest_framework.request import Request
```
Comment (beniwohli) [inline elasticapm/contrib/django/client.py:40]: could we rename this to `DrfRequest` to make it a bit clearer against what kind of request class we're making an instance check?

```suggestion
    from rest_framework.request import Request as DrfRequest

File: elasticapm/contrib/django/client.py [hunk 2]
```diff
@@ -36,6 +36,10 @@
...
+except ImportError:
+    Request = HttpRequest
```
Comment (beniwohli) [inline elasticapm/contrib/django/client.py:42]:
```suggestion
    DrfRequest = HttpRequest

**Review by beniwohli — CHANGES_REQUESTED** [review]
Great catch and nice fix! Just a small nitpick about naming :)

**Comment by beniwohli** [comment]
@Niccolum if you have a second, could you have a look at our CLA and sign it if you agree? That'll allow me to merge your PR :)

https://www.elastic.co/contributor-agreement

**Comment by Niccolum** [comment]
> @Niccolum if you have a second, could you have a look at our CLA and sign it if you agree? That'll allow me to merge your PR :)
> 
> https://www.elastic.co/contributor-agreement

Signed it. Hope, i did it correct )

**Comment by beniwohli** [comment]
@Niccolum sorry about the hassle, but could you sign the CLA again using the email you used for the commits? The one matching `n...v@e....com`

**Comment by Niccolum** [comment]
> @Niccolum sorry about the hassle, but could you sign the CLA again using the email you used for the commits? The one matching `n...v@e....com`

@beniwohli Oh, sorry for this
Is it okay?
Снимок экрана от 2021-04-27 14-56-49

**Comment by beniwohli** [comment]
@Niccolum yes, perfect, thanks! I'll merge the PR in a bit!

**Pull request merged**